docs/tutorials/introduction.ipynb

```diff
@@ -72,50 +77,60 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['alice'] = sc.Variable(dims=['row'], values=[1.0,1.1,1.2],\n",
-    "                         variances=[0.01,0.01,0.02], unit=sc.units.m)\n",
-    "sc.table(d)"
+    "ds['alice'] = sc.array(\n",
+    "    dims=['row'], values=[1.0, 1.1, 1.2], variances=[0.01, 0.01, 0.02], unit='m'\n",
+    ")\n",
+    "ds"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The column for `'alice'` contains two sub-columns with values and associated variances (uncertainties).\n",
+    "Under the hood, the column for `'alice'` contains two sub-columns with values and associated variances (uncertainties).\n",
     "The uncertainties are optional.\n",
+    "The datatype (`dtype`) is derived from the provided data, so passing `np.arange(3)` will yield a variable (column) containing 64-bit integers.\n",
     "\n",
-    "The datatype (`dtype`) is derived from the provided data, so passing `np.arange(3)` will yield a variable (column) containing 64-bit integers."
+    "As this dataset is 1-D we can visualize it as a table:"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "For many practicle purposes we want to associate a set of values (optionally a unit and variances also) with our dimension. Lets introduce a coordinate for `row` so that we can assign a row number starting at zero."
+    "sc.table(ds)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "d.coords['row'] = sc.Variable(dims=['row'], values=np.arange(3))\n",
-    "sc.table(d)"
+    "For many practicle purposes we want to associate a set of values (optionally a unit) with our dimension.\n",
```

Comment:
```suggestion
    "For many practical purposes we want to associate a set of values (optionally a unit) with our dimension.\n",
```
Not 100% sure this word is used correctly or if it is a translation from German.